README.md

```diff
@@ -280,12 +219,12 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
+
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+_NOTE: we recommend using the [MIT license](https://github.com/RolandAmini/hello--word/blob/feature/MIT.md) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] As the previous code reviewer mentioned related to customizing your `README`, please 🙏🏻  add the `MIT` license properly because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 . 

- [ ] Please remove the **Note** from your `MIT` license sections.